lib/containers/common.pm

```diff
@@ -43,7 +43,8 @@ sub install_podman_when_needed {
             assert_script_run "apt-get -y install podman", timeout => 220;
         }
         else {
-            add_suseconnect_product('sle-module-containers') if (is_sle '>=15');
+            # We may run openSUSE with DISTRI=sle and opensuse doesn't have SUSEConnect
+            add_suseconnect_product('sle-module-containers') if (check_os_release('SUSE Linux Enterprise', 'PRETTY_NAME') && is_sle('>=15'));
```

Comment:
Don't you already have `$host_os` here?

Reply:
What do you mean by that?

Reply:
Yes, but `$host_os` was used only for `ubuntu` and `centos` before and so it was empty for SLE and openSUSE. Now it should have always a value.